docs/src/pages/[platform]/getting-started/installation/dependencies.mdx

```diff
@@ -0,0 +1,185 @@
+  <Tabs.Item value="expo">Expo</Tabs.Item>
+  </Tabs.List>
+  <Tabs.Panel value="react-native">
+
+  Install `@aws-amplify/ui-react-native` and its dependencies with [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/):
+
+  <InstallScripts framework="react-native" />
+
+  If your project will support Federated Sign In using the React Native `Authenticator` the `@aws-amplify/rtn-web-browser` package is also required:
+
+  <InstallScripts command="npm install @aws-amplify/rtn-web-browser" framework="react-native" />
+
+  `@aws-amplify/react-native` and `@aws-amplify/rtn-web-browser` require a minimum iOS deployment target of `13.0`, open the _Podfile_ located in the _ios_ directory and update the `target` value:
+
+  ```diff
+  - platform :ios, min_ios_version_supported
+  + platform :ios, 13.0
+  ```
+
+  Then install the iOS cocoapods, by running:
+
+  <InstallScripts command="npx pod-install" framework="react-native" />
+
+  The AWS SDK requires React Native applications to polyfill `crypto.getRandomValues` and `URL`. Import the [`react-native-get-random-values`](https://github.com/LinusU/react-native-get-random-values) and [`react-native-url-polyfill`](https://github.com/charpeni/react-native-url-polyfill) polyfills and them to the top of your application's root entry point (in most React Native apps this will be the top level `index.js`).
+
+  <Example>
+  <ExampleCode>
+  ```jsx
+  // index.js
+  import 'react-native-url-polyfill/auto';
+  import 'react-native-get-random-values';
+
+  import { AppRegistry } from 'react-native';
+  import App from './App';
+  import { name as appName } from './app.json';
+
+  AppRegistry.registerComponent(appName, () => App);
+  ```
+  </ExampleCode>
+  </Example>
+
+  </Tabs.Panel>
+  <Tabs.Panel value="expo">
+
+  Add `@aws-amplify/ui-react-native` and its dependencies using the [Expo CLI](https://expo.dev/tools#cli) to ensure installation of native module dependency versions compatible with your Expo app:
+
+  <br />
+  <TerminalCommand command="npx expo install @aws-amplify/ui-react-native aws-amplify @aws-amplify/react-native react-native-safe-area-context @react-native-async-storage/async-storage @react-native-community/netinfo" />
+  <br />
+
+  If your project will support Federated Sign In using the React Native `Authenticator` the `@aws-amplify/rtn-web-browser` package is also required:
+
+  <br />
+  <TerminalCommand command="npx expo install @aws-amplify/rtn-web-browser" />
+
+  </Tabs.Panel>
+  </Tabs.Container>
+</InlineFilter>
+
+<InlineFilter filters={['react']}>
+  Install `@aws-amplify/ui-react` with [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/):
+
+  <InstallScripts framework="react" />
+
+  After adding the `aws-amplify` and `@aws-amplify/ui-react` dependencies you are now ready to add any of our [components](../components) to your application.
+</InlineFilter>
+
+<InlineFilter filters={['swift']}>
+  Add info here on how to install FaceLivenessDetector for Swift.
```

Comment:
Know this line copy pasta but it reads as a todo, so i looked in to whether it is actually currently rendered or not and unable to find it on the current docs (might just be me missing it). How are we verifying whether theses changes are going to lead to any content updates?

Reply:
You are correct that this particular line is never rendered.  The getting-started instillation section doesn't have `swift` as one of its supported platforms, but there were swift files that would have appeared if swift was ever added as a supported platform so to be consistent I moved it into this file under a filter that will currently just never display.
To your larger question of how these changes are being validated, I don't have a good answer for that.  I've been manually validating that content isn't changing but I know that is not ideal and can be very error prone.  I'm open to suggestions for validating this content.